pyproject.toml

```diff
@@ -169,6 +171,7 @@ USE_IOMP = "0"
 # Import discovery
 ignore_missing_imports = true
 exclude = "(dist|docs|requirements|.*egg-info)/"
+files = ["lightning_uq_box/models"]   
```

Comment:
Could expand this to everything except `uq_methods`, that should be the only thing left that has issues.